cleanenv_test.go

```diff
@@ -730,7 +735,7 @@ func TestParseFileEnv(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			tmpFile, err := ioutil.TempFile(os.TempDir(), "*.env")
+			tmpFile, err := os.CreateTemp(os.TempDir(), "*.env")
```

Comment:
Please do not replace `ioutil.TempFile` with `os.CreateTemp` yet, since I want the libraty to be compatible with older Golang versions (in this case with 1.16) for a bit longer.